plugins/ledger/ledger.py

```diff
@@ -321,47 +315,38 @@ def sign_transaction(self, tx, password):
             # Get trusted inputs from the original transactions
             for utxo in inputs:
                 sequence = int_to_hex(utxo[5], 4)
-                if not p2shTransaction:
-                    txtmp = bitcoinTransaction(bfh(utxo[0]))
-                    trustedInput = self.get_client().getTrustedInput(txtmp, utxo[1])
-                    trustedInput['sequence'] = sequence
-                    chipInputs.append(trustedInput)
-                    redeemScripts.append(txtmp.outputs[utxo[1]].script)
-                else:
-                    tmp = bfh(utxo[3])[::-1]
-                    tmp += bfh(int_to_hex(utxo[1], 4))
-                    chipInputs.append({'value' : tmp, 'sequence' : sequence})
-                    redeemScripts.append(bfh(utxo[2]))
+                txtmp = bitcoinTransaction(bfh(utxo[0]))
```

Comment:
Does this remove support multi-signature and other P2SH transactions using the Nano S?

Reply:
I've only tested 2 cosigner multisig, but it should (since all transactions are BIP143, unlike core where its only segwit)